numpy/random/mtrand/mtrand.pyx

```diff
@@ -127,6 +127,7 @@ cdef extern from "initarray.h":
 # Initialize numpy
 import_array()
 
+import cython
```

Comment:
Does `cimport cython` work? That is what the cython documentation uses for examples.